Stuck "Loading.." when WeakAura refresh happens to fail

**Describe the bug**
If refreshing weakauras fails, the refresh button is disabled and the message at the center of the screen stays "Loading... Currently parsing retail WeakAuras" and the user can't retry the refresh operation. 
I think this was a intermittent problem with the wago.io api and not due to local savedvariables as this has worked before. Restarting the client and pressing refresh again I did not encounter the error again.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to `My WeakAuras`
2. Click on `Refresh`
3. See the "Failed to parse WeakAuras" error.

**Expected behavior**
If there is some intermittent problem with calling the wago.io api, causing the refresh operation to fail, the user should not end up in a deadlocked state, unable to retry the operation.

**Screenshots**
![unknown](https://user-images.githubusercontent.com/68285419/105019206-23d05b00-5a46-11eb-842a-4a7252dd98ff.png)

**Software involved**
Please complete the following information:
 - OS: Windows 10
 - Ajour version: 0.6.3

**Log Output**
10:53:15.634 [ajour::gui::update][DEBUG] Interaction::Refresh(My WeakAuras)
10:53:27.884 [ajour][ERROR] Failed to parse WeakAuras
10:53:27.884 [ajour][ERROR] caused by: expected value at line 1 column 1

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -32,6 +32,10 @@ and `Removed`.
 - Catalog fuzzy matching better removes unrelated results while retaining relevant
   results
 
+### Fixed
+
+- Refresh button in My WeakAura is now interactable at all times.
+
 ## [0.6.3] - 2021-01-14
 
 ### Added

--- a/src/gui/element/my_weakauras.rs
+++ b/src/gui/element/my_weakauras.rs
@@ -33,7 +33,6 @@ pub fn menu_container<'a>(
     accounts: &'a [String],
     chosen_account: Option<String>,
 ) -> Container<'a, Message> {
-    //
     // MyWeakAuras state.
     let state = state
         .get(&Mode::MyWeakAuras(flavor))
@@ -41,7 +40,7 @@ pub fn menu_container<'a>(
         .unwrap_or_default();
 
     // A row contain general settings.
-    let mut row = Row::new().height(Length::Units(35));
+    let mut row = Row::new().align_items(Align::Center);
 
     let mut update_all_button = Button::new(
         update_all_button_state,
@@ -49,10 +48,11 @@ pub fn menu_container<'a>(
     )
     .style(style::DefaultButton(color_palette));
 
-    let mut refresh_button = Button::new(
+    let refresh_button = Button::new(
         refresh_button_state,
         Text::new(localized_string("refresh")).size(DEFAULT_FONT_SIZE),
     )
+    .on_press(Interaction::Refresh(Mode::MyWeakAuras(flavor)))
     .style(style::DefaultButton(color_palette));
 
     let pick_list = PickList::new(
@@ -68,10 +68,6 @@ pub fn menu_container<'a>(
     if updates_available && !is_updating && !updates_queued {
         update_all_button =
             update_all_button.on_press(Interaction::UpdateAll(Mode::MyWeakAuras(flavor)));
-    }
-
-    if state == State::Ready {
-        refresh_button = refresh_button.on_press(Interaction::Refresh(Mode::MyWeakAuras(flavor)));
     }
 
     let update_all_button: Element<Interaction> = update_all_button.into();
@@ -123,8 +119,9 @@ pub fn menu_container<'a>(
 
     // Add space above settings_row.
     let settings_column = Column::new()
-        .push(Space::new(Length::Units(0), Length::Units(5)))
-        .push(row);
+        .push(Space::new(Length::Units(0), Length::Units(7)))
+        .push(row)
+        .push(Space::new(Length::Units(0), Length::Units(8)));
 
     // Wraps it in a container.
     Container::new(settings_column)

